src/ansys/sherlock/core/lifecycle.py

```diff
@@ -52,23 +53,32 @@ def __init__(self, channel):
         self.STEP_TYPE_LIST = ["RAMP", "HOLD"]
 
     def _init_cycle_types(self):
-        """Initialize the list for cycle types."""
+        """Initialize the list for cycle types.
+
+        Available Since: 2021R1

```

Comment:
Do you think there is value in documenting this for non-public methods?

Reply:
No